webclient/webclient.js

```diff
@@ -158,7 +158,7 @@ var WebClient = {
       "password" : this.options.pass,
       "clientid" : this.guid(),
       "clientver" : "webclient-0.1 (2015-12-23)",
-      "clientfeatures" : [ "client_id", "client_ver"],
+      "clientfeatures" : [ "client_id", "client_ver", "client_warnings", "feature_set", "mod_log_lookup", "room_chat_history", "user_ban_history"]
```

Comment:
i had to add them in order to make the client able to connect to wooger's server, that requires some of those  features in clients (i just added them all, didn't check them one by one).